doc/user/content/manage/user-management/rbac-newrole.md

```diff
@@ -185,38 +186,152 @@ to a user in your Materialize organization.
 1. To review the permissions a role has, you can view the object data:
 
    ```sql
-   SELECT name, privileges::text FROM mz_tables WHERE name='dev_table';
+   SELECT name, privileges FROM mz_tables WHERE name='dev_table';
    ```
 
-   The output should return the owner, the level of permission, and the name of
-   the role on the table.
+   The output should return the owner object ID, the level of permission, and the owner ID of the role on the table.
```

Comment:
You've specified owner ID twice here, did you only mean to include it once? Also you will need to add the `owner_id` column to your query to see the owner of a table.